### PR TITLE
Grafana Zabbix plugin support added

### PR DESCRIPTION
**Pull request by dmajere**
Grafana Zabbix pluging support added.
Plugin docs: http://docs.grafana-zabbix.org/

### CODE REVIEW

File: grafanalib/zabbix.py [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+
+def is_valid_interval(instance, attribute, value):
+    """
+    A validator that raises a :exc:`ValueError` if the attribute value is not
+    matching on regular expression.
+    """
+    if not re.match("^[+-]?\d*[smhdMY]$", value):
+        raise ValueError(
+            "valid interval should be a string "
+            "matching an expressiong: ^[+-]?\d*[smhdMY]$. "
```
Comment (jml) [inline grafanalib/zabbix.py]: expression

File: grafanalib/zabbix.py [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+            "Exaxples: 24h 7d 1M +24h -24h")
```
Comment (jml) [inline grafanalib/zabbix.py]: Examples:

File: grafanalib/zabbix.py [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+
+
+def is_number(instance, attribute, value):
+    """
+    A validator that raises a :exc:`ValueError` if the attribute value is not
+    a number (not int or float).
+    """
+    if isinstance(value, int) or isinstance(value, float):
+        return
+    raise ValueError("{attr} should be interger or float".format(
```
Comment (jml) [inline grafanalib/zabbix.py]: integer

File: grafanalib/zabbix.py [hunk 4]
```diff
@@ -0,0 +1,706 @@
...
+            "matching an expressiong: ^[+-]?\d*[smhdMY]$. "
+            "Exaxples: 24h 7d 1M +24h -24h")
+
+
+def is_number(instance, attribute, value):
+    """
+    A validator that raises a :exc:`ValueError` if the attribute value is not
+    a number (not int or float).
+    """
+    if isinstance(value, int) or isinstance(value, float):
```
Comment (jml) [inline grafanalib/zabbix.py]: This function probably isn't necessary. Can just do `instance_of(Number)`, see https://docs.python.org/2/library/numbers.html#numbers.Number for details.

File: grafanalib/zabbix.py [hunk 5]
```diff
@@ -0,0 +1,706 @@
...
+    """ZabbixGroupByFunction
+
+    Takes each timeseries and consolidate its points falled in given interval
+    into one point using function, which can be one of: avg, min, max, median.
+    http://docs.grafana-zabbix.org/reference/functions/#groupBy
+    """
+
+    __options = ("avg", "min", "max", "median")
+    __default_interval = "1m"
+    __default_function = "avg"
```
Comment (jml) [inline grafanalib/zabbix.py]: Please don't use double underscores for privacy. Please just use a single underscore.

File: grafanalib/zabbix.py [hunk 6]
```diff
@@ -0,0 +1,706 @@
...
+
+@attr.s
+class ZabbixScaleFunction(object):
+    """ZabbixScaleFunction
+
+    Takes timeseries and multiplies each point by the given factor.
+    http://docs.grafana-zabbix.org/reference/functions/#scale
+    """
+
+    __default_factor = 100
```
Comment (jml) [inline grafanalib/zabbix.py]: Same here.

**Comment by dmajere** [comment]
Fixed according to @jml comments

**Pull request merged**